### DISCUSSION TITLE
httpx not working | No Results showing

### DISCUSSION THREAD

**Discussion by IamLucif3r** [projectdiscovery/httpx#630] [General]
### Current Behavior:  

When I type any command (e.g. cat domains.txt | httpx ), it ends up without giving any result. You can see the screenshot.
![Screenshot at 2022-05-18 18-09-36](https://user-images.githubusercontent.com/53969258/169040551-b286565e-5fa3-4e2a-ae03-ed57c39d2624.png)

### Expected Behavior:

It must return the results.

### Steps To Reproduce:

1. Run httpx with a list of subdomains. 
2. Use command > cat domains.txt | httpx

### Anything else:

No.

**Comment by ehsandeep** [comment]
@IamLucif3r Please attach `domains.txt` or example input to verify this behavior.

**Comment by mrarafatstrange** [comment]
I am facing this respective problem, can anyone please help me by providing the commands?
httpx says DBG Failed..

**Comment by olearycrew** [comment]
I'm unable to reproduce this behavior - is it possible the domains.txt contains domains with not HTTP server running?

<img width="584" alt="CleanShot 2023-02-07 at 10 21 19@2x" src="https://user-images.githubusercontent.com/6044920/217286452-9eb3d06c-28c9-4f9f-8805-53bfac4873db.png">
